### DISCUSSION TITLE
Have a problem when i start new app ten

### DISCUSSION THREAD

**Discussion by DDark121** [rio-labs/rio#181] [Q&A]
![photo_2024-11-26_10-24-09](https://github.com/user-attachments/assets/30daac5b-3087-44ba-ac87-876e0fced038)
when i start new app project he does't run.

I use visual studio. Website project working without any problem

**Comment by Aran-Fey** [comment]
I suspect you haven't installed the necessary dependencies. Does running `pip install rio-ui[window]` fix it?

**Reply by DDark121** [reply]
Its working thanks. Rio is amazing tool
